app/blueprints/round/routes.py

```diff
@@ -90,9 +90,7 @@ def create_round():
     fund = get_fund_by_id(fund_id)
     if form.validate_on_submit():
         new_round = create_new_round(form)
-        flash_message(
-            message="New application created successfully", href="#", href_display_name=new_round.title_json["en"]
-        )
+        flash_message(message="New application created", href="#", href_display_name=new_round.title_json["en"])
```

Comment:
I think we can show the application details page link over here 
```suggestion
        flash_message(
            message="New application created successfully", href=url_for("round_bp.round_details", round_id=new_round.round_id), href_display_name=new_round.title_json["en"]
        )
```